### DISCUSSION TITLE
Install stuck

### DISCUSSION THREAD

**Discussion by bishop340** [Security-Onion-Solutions/securityonion#3344] [Unsupported Versions]
Make it all the way through the menus and then it sticks at 0% on running initial configuration steps. Let it run all night and it never started. Restarts the vm and this is what my sosetuplog has

<img width="1026" alt="Screen Shot 2021-03-08 at 10 21 53 AM" src="https://user-images.githubusercontent.com/80270237/110341607-82eb2e80-7ff8-11eb-87e9-e8bc51a4dc14.png">
<img width="1026" alt="Screen Shot 2021-03-08 at 10 22 34 AM" src="https://user-images.githubusercontent.com/80270237/110341621-87afe280-7ff8-11eb-8f84-c2e231d23c5e.png">

**Comment by Doshin-the-Giant** [comment]
I'm having the same issue. If anyone knows the resolution to this it would be appreciated.

**Comment by defensivedepth** [comment]
Looks like an ISO install. Have you verified the ISO to confirm that it is not corrupted?

https://github.com/Security-Onion-Solutions/securityonion/blob/master/VERIFY_ISO.md
